vms/shift/views.py

```diff
@@ -192,6 +194,9 @@ def cancel(request, shift_id, volunteer_id):
             try:
                 cancel_shift_registration(volunteer_id, shift_id)
                 if admin:
+                    vol_email = Volunteer.objects.get(pk=volunteer_id).email
+                    send_mail("Shift Cancelled", "Your shift has been cancelled",
+                              "messanger@localhost.com", [vol_email], fail_silently=False)

```

Comment:
Change it to `email_host'. And, how would you authenticate password at SMTP servers?

Reply:
what do you mean by changing to email_host?

Reply:
We can authenticate passwords by encrypting it and return the original password to a variable. https://stackoverflow.com/questions/42030080/encypting-smtp-server-password-in-django-settings

Reply:
I was talking about the `email_host` variable in settings.py file. In this way, the admin will be able to select the email id dynamically. What's your say on this?